<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaixaForm</class>
 <widget class="QWidget" name="CaixaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#header {
	font-size: 18pt;
	font-weight: bold;
}

QDoubleSpinBox, QLineEdit {
	font-size:14pt;
}

QGroupBox {
	font-size: 14pt;
}

QGroupBox::title {
	font-size:14pt;
	font-weight: bold;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="0" column="0">
    <widget class="QLabel" name="header">
     <property name="text">
      <string>Caixa 01 - Aberto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="omarket.qrc">:/icon/128/caixa.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>CÓDIGO DE BARRAS:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLineEdit" name="barcode"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>QUANTIDADE:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="quantidade">
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2" rowspan="3">
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>VALOR TOTAL DA COMPRA:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QDoubleSpinBox" name="v_total">
          <property name="prefix">
           <string>R$ </string>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>TOTAL RECEBIDO:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="r_total">
            <property name="prefix">
             <string>R$ </string>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>TROCO:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="troco">
            <property name="prefix">
             <string>R$ </string>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>FINALIZAR E IMPRIMIR (CTRL+P)</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+P</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>ADICIONAR (ENTER)</string>
         </property>
         <property name="shortcut">
          <string>Enter</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>VALOR UNITÁRIO:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QDoubleSpinBox" name="valor">
          <property name="prefix">
           <string>R$ </string>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>VALOR TOTAL:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QDoubleSpinBox" name="p_total">
          <property name="prefix">
           <string>R$ </string>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>184</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>barcode</tabstop>
  <tabstop>quantidade</tabstop>
  <tabstop>r_total</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>valor</tabstop>
  <tabstop>p_total</tabstop>
  <tabstop>v_total</tabstop>
  <tabstop>troco</tabstop>
 </tabstops>
 <resources>
  <include location="omarket.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>barcode</sender>
   <signal>editingFinished()</signal>
   <receiver>CaixaForm</receiver>
   <slot>barcodeReaded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CaixaForm</receiver>
   <slot>addItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CaixaForm</receiver>
   <slot>print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quantidade</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CaixaForm</receiver>
   <slot>changeQuantidade(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>r_total</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CaixaForm</receiver>
   <slot>calcular(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>barcodeReaded()</slot>
  <slot>addItem()</slot>
  <slot>print()</slot>
  <slot>changeQuantidade(double)</slot>
  <slot>calcular(double)</slot>
 </slots>
</ui>
